conf/parser.py

```diff
@@ -47,8 +47,8 @@ def __init__(self, fname):
         self.core_ifname = None
         self.interfaces = dict()
         self.enable_ntf = False
-        self.notify_sockaddr = "/tmp/notifycp"
-        self.endmarker_sockaddr = "/tmp/pfcpport"
+        self.notify_sockaddr = "/pod-share/notifycp"
+        self.endmarker_sockaddr = "/pod-share/pfcpport"
```

Comment:
keep these as /tmp and override in helm chart

Reply:
other places too

Reply:
done